code/workspaces/web-app/src/reducers/dataStorageReducer.spec.js

```diff
@@ -24,39 +25,79 @@ describe('dataStorageReducer', () => {
     expect(dataStorageReducer(undefined, {})).toEqual({ fetching: false, value: [], error: null });
   });
 
-  it('should handle LOAD_DATASTORAGE_PENDING', () => {
+  describe('LOAD_DATASTORAGE_ACTION', () => {
+    it('should handle LOAD_DATASTORAGE_PENDING', () => {
     // Arrange
-    const type = `${LOAD_DATASTORAGE_ACTION}_${PROMISE_TYPE_PENDING}`;
-    const action = { type };
+      const type = `${LOAD_DATASTORAGE_ACTION}_${PROMISE_TYPE_PENDING}`;
+      const action = { type };
```

Comment:
Does the comment also need to be indented? This is the same for other tests in this file too.

Reply:
Good spot